code/modules/power/cell.dm

```diff
@@ -43,6 +43,8 @@
 		return 0
 	if(charge < amount)	return 0
 	charge = (charge - amount)
+	if(!istype(loc, /obj/machinery/power/apc))
+		feedback_add_details("cell_used","[src.name]")

```

Comment:
Recommend you replace with type or some other identifier instead of name, name is often changed by admins or code